.github/workflows/prcodecheck.yml

```diff
@@ -49,6 +49,21 @@ jobs:
           sudo make install
         working-directory: kpsr-core
 
+      - name: Clone kpsr-codegen
+        uses: actions/checkout@v3
+        with:
+          repository: klepsydra-technologies/kpsr-codegen
+          ref: main
+          submodules: recursive
+          path: kpsr-codegen
+
+      - name: Build and install kpsr-codegen
+        run: |
+          pip install build
+          python3 -m build --outdir dist
+          pip3 install dist/kpsr_codegen-1.0-py3-none-any.whl
```

Comment:
In line 62 there is jus `pip` but here we have `pip3` in many cases `pip` probably will point to version 3 but it should be consistent.
Same situation for the rest of CI files.

Reply:
Done, changed `pip` to `pip3`